examples/mp-openapi/src/main/java/org/superbiz/openapi/Weather.java

```diff
@@ -0,0 +1,25 @@
+package org.superbiz.openapi;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+@Schema(name="Weather", description="POJO that represents weather.")

```

Comment:
Can you follow a pattern with spaces? name="Wather" has no space between name and the value. But for @Schema the variables have space. Can you make sure to format them correctly having one space between the variable and the value?